LibraryApp/src/redux/auth/reducer.js

```diff
@@ -5,15 +5,28 @@ import { actions } from './actions';
 const initialState = {
   loading: false,
   logged: false,
-  authError: null
+  authError: null,
+  initialAuthLoading: false,
+  email: '',
+  password: ''
 };
 
 const reducer = (state = immutable(initialState), action) => {
+  const { payload } = action;
   switch (action.type) {
-    case actions.INIT_STORED_USER:
+    case actions.AUTH_INIT:
+      return state.merge({
+        initialAuthLoading: true
+      });
+    case actions.AUTH_INIT_SUCCESS:
```

Comment:
You should use redux recompose here to avoid this

Reply:
Should I do it even if i don't need initialAuth or initialAuthError variables?